package.json

```diff
@@ -57,7 +57,7 @@
     "source-map-support": "0.5.6",
     "supertest": "3.4.2",
     "tslint": "5.9.1",
-    "typescript": "^2.8.3"
+    "typescript": "^3.4.5"
```

Comment:
I would maybe put this version bump in a separate commit.

Reply:
I'll take a look, probably I need this change for some language feature